themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -34,7 +34,34 @@
                   <h2>
                     <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull _record_link"  data-view="<?=$this->escapeHtmlAttr($this->params->getOptions()->getListViewOption())?>">
                     <?=$item['Data']?> </a>
-                  </h2>
+		  </h2>
+		<?php elseif ('Au' == $item['Group']): ?>
+                    <?php if (isset($this->config->AuthorResultListInfo->ResultListPage) && ($this->config->AuthorResultListInfo->ResultListPage == 'Short')): ?>
+                        <?php $authors = $driver->getPrimaryAuthorsWithHighlighting(); ?>
+			<?php $authorCount = count($authors) ?>
+		        <?php $authorNameLimit = $this->config->AuthorResultListInfo->AuthorNameLimit ?? 3; ?>
+                        <?php if (isset($authors) && !empty($authors)): ?>
+                            <b><?=$this->transEsc($item['Label'])?>:</b>
+                            <?php $count = 0; ?>
+                            <?php foreach ($authors as $author): ?>
+                                <a href="<?=$this->record($driver)->getLink('author', $this->highlight($author, null, true, false))?>"><?=$this->highlight($author)?></a>
+                                <?php $count++?>
+                                <?php if ($authorCount == $count): ?>
+                                    <?php break; ?>
+                                <?php elseif ($count > $authorNameLimit): ?>
+                                    <?=$this->transEsc('more_authors_abbrev')?>
+				    <?php break; ?>
+                                <?php else:?>
+                                    ,
```

Comment:
Does this syntax create unwanted spaces in front of the commas? I've seen that kind of problem in the past, and it seems like it might be an issue here. If that's the case, you might want to move more of the code into PHP-land and use an echo statement if you need to prevent the code from outputting unwanted whitespace.

Reply:
thanks @demiankatz , yes it does. I've switched to using an echo statement in my follow-up. I wasn't sure if I needed to wrap the comma in a $this->transEsc(','); so I added that just in case.